src/core/cita_bft.rs

```diff
@@ -1289,15 +1283,18 @@ impl Bft {
 
             let mut flag = false;
 
-            for &(ref height, ref blocktxs) in &self.block_txs {
+            for &mut (height, ref mut blocktxs) in &mut self.block_txs {
                 trace!(
                     "new_proposal BLOCKTXS get height {}, self height {}",
-                    *height,
+                    height,
                     self.height
                 );
-                if *height == self.height - 1 {
+                if height == self.height - 1 {
                     flag = true;
-                    block.set_body(blocktxs.get_body().clone());
+                    // Take all transactions in case of the proposal include them can't be verified
```

Comment:
Need to add the explanation that it must be `take` if compact-block-relay. If it's a full-block-relay proposal either `take` or `get` is ok.

Reply:
There are some grammar errors in the first sentence.

Reply:
I will change this sentence, but I do not think it has grammar errors.